g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/conf/PropertyConfigs.java

```diff
@@ -197,4 +200,8 @@ public String getSyncListPath() {
 	public int getSyncBatchSize() { return (syncBatchSize - 1); }
 
 	public boolean isSyncBatchEnable() { return syncBatchEnable; }
+
+	public int getSyncReqBodySize(){
+		return  (1073741824*Integer.valueOf(this.reqBodySizeStr.toUpperCase().replaceAll("M", "").trim()))-536870912;
```

Comment:
Looks like the multiple 1073741824 is not right, should be 1048576(1024*1024), and suggest write it as 1024*1024, which is more readable. And same comment with 536870912.

Reply:
done